config/api.yml

```diff
@@ -1597,3 +1597,30 @@
       :delete:
       - :name: delete
         :identifier: zone_delete
+  :virtual_templates:
+    :description: Virtual Templates
+    :identifier: virtual_template
```

Comment:
@imtayadeway Ok, I can update that. Is there a reason behind that naming convention? I tried to find a pattern from the yaml but didn't pick up on that